Tanks/Assets/Scripts/Camera/CameraControl.cs

```diff
@@ -2,50 +2,45 @@
 
 public class CameraControl : MonoBehaviour
 {
-    public float m_DampTime = 0.2f;                 
-    public float m_ScreenEdgeBuffer = 4f;           
-    public float m_MinSize = 6.5f;                  
-    [HideInInspector] public Transform[] m_Targets; 
-
-
-    private Camera m_Camera;                        
-    private float m_ZoomSpeed;                      
-    private Vector3 m_MoveVelocity;                 
-    private Vector3 m_DesiredPosition;              
+    [SerializeField] private const float dampTime = 0.2f;
+    [SerializeField] private const float screenEdgeBuffer = 4f;
+    [SerializeField] private const float minSize = 6.5f;                  
+    [HideInInspector] public Transform[] targets; 
```

Comment:
This is fine. In our codebase the header will always be on a line of its own above the line it's heading though, just so you know